
// try catch with locals

f:FN(ER(VD))${() '"This is an error"; }

fx:FN(f;VD)${(x) [e]'x(); $(FD;1) << "Error f called\n" }

//f()

fx(f)
